<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QAction{color:white;}
QMainWindow{color:white;background-color:black;}
</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPatch_Light">
    <property name="title">
     <string>Patch</string>
    </property>
    <addaction name="patchButton"/>
   </widget>
   <widget class="QMenu" name="menuEffects">
    <property name="title">
     <string>Effects</string>
    </property>
    <addaction name="effectsButton"/>
    <addaction name="selectButton"/>
    <addaction name="selectAllLightsButton"/>
    <addaction name="chooseColourButton"/>
   </widget>
   <widget class="QMenu" name="menuRig">
    <property name="title">
     <string>Rig</string>
    </property>
    <addaction name="openButton"/>
    <addaction name="saveButton"/>
    <addaction name="defaultSaveButton"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <addaction name="recordPlayback"/>
    <addaction name="playbackButton"/>
   </widget>
   <widget class="QMenu" name="menuSequence">
    <property name="title">
     <string>Sequence</string>
    </property>
    <addaction name="sequenceButton"/>
    <addaction name="openSequenceButton"/>
    <addaction name="stopSequenceButton"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="snakeButton"/>
   </widget>
   <addaction name="menuPatch_Light"/>
   <addaction name="menuEffects"/>
   <addaction name="menuRig"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuSequence"/>
   <addaction name="menuGame"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="patchButton">
   <property name="text">
    <string>Patch New Light</string>
   </property>
  </action>
  <action name="effectsButton">
   <property name="text">
    <string>Effects Window</string>
   </property>
  </action>
  <action name="selectButton">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select Lights</string>
   </property>
  </action>
  <action name="openButton">
   <property name="text">
    <string>Open Rig</string>
   </property>
  </action>
  <action name="saveButton">
   <property name="text">
    <string>Save Rig</string>
   </property>
  </action>
  <action name="defaultSaveButton">
   <property name="text">
    <string>Auto Save Rig</string>
   </property>
  </action>
  <action name="recordPlayback">
   <property name="text">
    <string>Record Playback</string>
   </property>
  </action>
  <action name="playbackButton">
   <property name="text">
    <string>Open Playback</string>
   </property>
  </action>
  <action name="sequenceButton">
   <property name="text">
    <string>Sequence Window</string>
   </property>
  </action>
  <action name="openSequenceButton">
   <property name="text">
    <string>Open Sequence</string>
   </property>
  </action>
  <action name="snakeButton">
   <property name="text">
    <string>Snake Game</string>
   </property>
  </action>
  <action name="chooseColourButton">
   <property name="text">
    <string>Choose Colour</string>
   </property>
  </action>
  <action name="selectAllLightsButton">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select All Lights</string>
   </property>
  </action>
  <action name="stopSequenceButton">
   <property name="text">
    <string>Stop Sequence</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
